index.html

```diff
@@ -36,6 +36,321 @@ <h5>LET’S CONNECT</h5>
              </ul>
             </div>
         </section>
+        <section class="Snapshoot">
+
+
+          <article class="article1">
+           <figure>
+            <img src="images/Snapshoot Portfolio.png" alt="Portfolio1">
+           </figure>
+
+
+              <div class="article1div">
+
+
+                  <h3 class="texttitle font1">Tonic</h3>
+
+
+                  <ul>
```

Comment:
-Kindly, Your cards are not merged well as the Figma design, please give it a margin-left .👍👌


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/212412245-4bc1bfe1-0321-4c50-9016-e2557306af82.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/212412240-8046f325-a922-4d12-9d0c-00b1b30f1d08.png" width="200">|


😎️

🤝️